apps/DesktopStreamer/Stream.cpp

```diff
@@ -56,21 +56,25 @@
 
 #define CURSOR_IMAGE_FILE     ":/cursor.png"
 #define CURSOR_IMAGE_SIZE     20
+#define CURSOR_TIMEOUT        1000 /*ms*/
 
 class Stream::Impl
 {
 public:
-    Impl( Stream& stream, const MainWindow& parent,
-          const QPersistentModelIndex window, const int pid )
-    : _stream( stream )
-    , _parent( parent )
-    , _window( window )
-    , _cursor( QImage( CURSOR_IMAGE_FILE ).scaled(
-                   CURSOR_IMAGE_SIZE * parent.devicePixelRatio(),
-                   CURSOR_IMAGE_SIZE * parent.devicePixelRatio(),
-                   Qt::KeepAspectRatio ))
-    , _pid( pid )
-    {}
+Impl( Stream& stream, const MainWindow& parent,
```

Comment:
indentation was correct before? The rest of this class is mostly wrong though...